compiler/aarch64/codegen/GenerateInstructions.hpp

```diff
@@ -1123,6 +1123,93 @@ TR::Instruction *generateVectorShiftImmediateInstruction(
                   uint32_t shiftAmount,
                   TR::Instruction *preced = NULL);
 
+/**
+ * @brief Generates duplicate vector element instruction
+ *
+ * @param[in] cg          : CodeGenerator
+ * @param[in] op          : opcode
+ * @param[in] node        : node
+ * @param[in] treg        : target register
+ * @param[in] sreg        : source register
+ * @param[in] srcIndex    : source element index
+ * @param[in] preced  : preceding instruction
```

Comment:
Please align the indentation of `:` with other lines.
The same comment for three other new function declarations.

Reply:
Fixed indentation of  `:`.